src/AspNetCoreController/Example/Startup.cs

```diff
@@ -29,8 +28,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddSystemTextJson()
                 .AddValidationRule<InputValidationRule>()
                 .AddGraphTypes(typeof(StarWarsSchema).Assembly)
-                .AddDocumentExecuter<ApolloTracingDocumentExecuter>()
-                .AddMetrics(_ => true, (provider, _) => provider.GetRequiredService<IOptions<GraphQLSettings>>().Value.EnableMetrics));
+                .AddMemoryCache()
+                .AddApolloTracing(_ => true));
```

Comment:
```suggestion
                .AddApolloTracing(options => options.RequestServices!.GetRequiredService<IOptions<GraphQLSettings>>().Value.EnableMetrics));
```
Let's retain the sample showing how it can be configured